python/ql/lib/semmle/python/ApiGraphs.qll

```diff
@@ -477,6 +700,12 @@ module API {
       not result instanceof DataFlow::ModuleVariableNode
     }
 
+    /**
+     * Holds if `rhs` is the right-hand side of a definition of node `nd`.
+     */
+    cached
+    predicate rhs(TApiNode nd, DataFlow::Node rhs) { nd = MkDef(rhs) }
```

Comment:
It could be inlined, but I think I would like to keep it as is.  

In JS the same predicate is slightly less trivial. And if you ever get module-exports nodes in Python, the the predicate will become slightly less trivial.

Reply:
I see. We should probably have it as a goal to be able to capture module exports, so I guess we can keep this.